website/views/__init__.py

```diff
@@ -23,22 +24,47 @@ def account_settings():
     return "To be implemented"
 
 
+triggers = ["When Motion is Detected", "When No Motion is Detected"]
+actors = ["Light Switch", "Thermostat", "Motion Sensor"]
 @app.route('/device/actions')
 def device_actions():
-    return render_template("deviceactions.html")
+    return render_template("deviceactions.html", triggers=triggers, actors=actors)
 
 
+class Devices:
```

Comment:
Please give this data structure a more meaningful name. What is it supposed to represent?